cmd/platform.go

```diff
@@ -122,14 +124,16 @@ func createGithubClient(flag *flag.FlagSet, verifyFlags bool, readOnly bool) (mu
 	orgs, _ := flag.GetStringSlice("org")
 	users, _ := flag.GetStringSlice("user")
 	repos, _ := flag.GetStringSlice("repo")
+	repoSearch, _ := flag.GetString("repo-search")
+	repoExclude, _ := flag.GetString("repo-exclude")
```

Comment:
You should Compile this regexp once using `MustCompile` (https://pkg.go.dev/regexp#MustCompile) and then reuse the compiled version of the regexp instead of calling `Match` on the string regexp (the latter is much slower).

Reply:
`MustCompile` is now used when creating the regular expression in `func (g *Github) getRepositories(ctx context.Context) ([]*github.Repository, error)`